verifiable-db/src/block_tree/mod.rs

```diff
@@ -12,6 +12,18 @@ use plonky2_ecdsa::gadgets::nonnative::CircuitBuilderNonNative;
 use plonky2_ecgfp5::gadgets::curve::{CircuitBuilderEcGFp5, CurveTarget};
 pub use public_inputs::PublicInputs;
 
+/// Common function to compute the digest of the block tree which uses a special format using
+/// scalar multiplication
+pub fn scalar_mul(
```

Comment:
Isn't this a duplicate of `compute_index_digest`? You wanted to rename it?

Reply:
Yeah it's a duplicate, i wanted to rename it but idk how it ended up there.

Reply:
So should remove one of the duplicate methods I think?